.github/workflows/publish-contracts.yml

```diff
@@ -23,14 +23,22 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
-      - name: Run the environment
-        run: npm run env:start-base
-
-      - name: Build image
-        run: ./scripts/build-image.sh
-
-      - name: Publish image
-        run: ./scripts/release-image.sh
+      # Commented out just for test
+      # - name: Run the environment
+      #   run: npm run env:start-base
+
+      # - name: Build image
+      #   run: ./scripts/build-image.sh
+
+      - name: Deploy to Goerli
```

Comment:
Deployment to the docker env is specific. It needs to be invoked inside the `build-image.sh` because it must be part of the image that is built there.